sdk/servicebus/Azure.Messaging.ServiceBus/src/Azure.Messaging.ServiceBus.csproj

```diff
@@ -47,6 +47,9 @@
     <Compile Include="$(AzureCoreSharedSources)PageResponseEnumerator.cs" Link="SharedSource\Azure.Core\PageResponseEnumerator.cs" />
 
   </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\core\Azure.Core.Experimental\src\Azure.Core.Experimental.csproj" />
```

Comment:
Might be worth copying from https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/search/Azure.Search.Documents/Directory.Build.props so you don't have to worry about flipping this back forth (or doing it for everyone up where the same setting for Azure.Core behavior lives).

Reply:
I guess a single switch for all projects might be nice, but I will defer to @pakrym as he is the one that updates to package refs after a release.

Reply:
I don't have a strong preference about how you do it but I also don't think the https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/search/Azure.Search.Documents/Directory.Build.props#L23 condition works in the current case.

Reply:
Most of my stuff is overkill to easily re/move functionality at the last minute, but I think having `UseProjectReferenceToAzureClients` to switch between package and projects like

```xml
  <!-- Add a reference to Azure.Core.Experimental -->
  <ItemGroup Condition="'$(IsTestProject)' != 'true'">
    <ProjectReference
      Condition="'$(UseProjectReferenceToAzureClients)' == 'true'"
      Include="$(MSBuildThisFileDirectory)..\..\..\core\Azure.Core.Experimental\src\Azure.Core.Experimental.csproj" />
    <PackageReference
      Condition="'$(UseProjectReferenceToAzureClients)' != 'true'"
      Include="Azure.Core.Experimental" />
  </ItemGroup>
```

would better here than an explicit reference.

Reply:
Wouldn't we need a different variable though? UseProjectReferenceToAzureClients is switched to true for the [Project_References](https://github.com/Azure/azure-sdk-for-net/blob/1c5556053861921701d59ed2fb4319dd7fa0fe1d/eng/pipelines/templates/jobs/archetype-sdk-client.yml#L122) CI/Tests, but this would need to be manually changed by @pakrym once the next release of Experimental goes out.